README.md

```diff
@@ -33,7 +33,9 @@ housekeeping:
     password: Ch4ll3ng3
 ```
 
-Note: To use MySQL and similar database systems, the schema specified in the configuration needs to exist, as the value for `housekeeping.data-source.url` needs to be a valid URI.
+Notes:
+* Some users have seen issues using the MySQL driver on AWS Aurora. [AWS recommends](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Connecting.html) to use the MariaDB driver for those use cases.
+* To use MySQL and similar database systems, the schema specified in the configuration needs to exist, as the value for `housekeeping.data-source.url` needs to be a valid URI.
```

Comment:
as the value -> and the value
[not your typo, but just noticed it]

Reply:
"as the value" was meant to be "because the value". Should I change "as" into "because"?

Reply:
Ah right, that makes more sense. Maybe do "configuration needs to exist (i.e. the value for `housekeeping.data-source.url` needs to be a valid URI).`

Reply:
Done